buildSrc/build.gradle

```diff
@@ -99,9 +101,11 @@ dependencies {
 
 // Gradle 2.14+ removed ProgressLogger(-Factory) classes from the public APIs
 // Use logging dependency instead
+// Gradle 4.3.1 stopped releasing the logging jars to jcenter, just use the last available one
+GradleVersion logVersion = GradleVersion.current() > GradleVersion.version('4.3') ? GradleVersion.version('4.3') : GradleVersion.current()
```

Comment:
I don’t think we should do this, I think we should depend on the right version. I understand that entails an additional external dependency, but the dependency will end up cached locally anyway except on Gradle upgrades. For CI this has no impact since we upgrade infrequently there.

Reply:
I think this is the wrong tradeoff. We use `gradle-logging` as a compileOnly dependency, and the interface we use has been super stable since forever. We would notice (build breaks) if something would change in that regard in future versions. Ignoring security concerns, adding a dependency on a repository that might not have the same uptime as jcenter could annoy us more... I've updated the PR according to your suggestion as I want to see this fixed.

Reply:
Okay, if this dependency is as stable as you say let's take it the way that you proposed first.